config/locales/en.yml

```diff
@@ -382,6 +382,7 @@ en:
     labels:
       activities:
         date: Date
+        opens_at: Signup from
```

Comment:
Although 'Signup from' is fine, I would like to propose two alternatives to see what you prefer.

"Enrollable from", the confirmation message when you signed up for an activity says "Succesfully _enrolled_ for this activity" and this is more consistent with that. I am not certain whether this is better.

"(Admissions) Open(s) at", not for a particular reason, just reads better imo.

Reply:
Also, shouldn't "open_at" be "open_from"? I don't know if that is more idiomatic

Reply:
Good point. I'll change the key to `open_from` and change the text to `Enrollable from`.

The button will be another PR, it isn't relevant here :)